frontend/src/lib/WebsocketConnection.tsx

```diff
@@ -483,6 +484,11 @@ export class WebsocketConnection {
   }
 }
 
+const StyledBashCode = styled.code({
+  content: '"$"',
+  marginRight: "1ex",
+})
```

Comment:
do you not need the ::before anymore? dont think this is testable with core previews :(